simulator/mips/mips_memory.h

```diff
@@ -14,7 +14,7 @@
 class MIPSMemory : private Memory
```

Comment:
I think "Memory" should be renamed to FuncMemory or BasicMemory to prevent ambiguities.